eva_sub_cli/validators/validator.py

```diff
@@ -86,17 +89,33 @@ def validate_and_report(self):
         self.report()
 
     def validate(self):
+        self.set_up_output_dir()
         self.verify_files_present()
         self._validate()
         self._collect_validation_workflow_results()
 
     def report(self):
-        self.create_reports()
+        report_path = self.create_reports()
         self.update_config_with_validation_result()
+        self.clean_up_output_dir(report_path)
```

Comment:
The report contains path to files that are moved in the `clean_up`. These path won't be correct anymore.

Reply:
Thanks for catching, this is fixed though the code is a bit more messy... maybe there's a nicer way to do it.